### DISCUSSION TITLE
Use same shortcut for more than one similar operation

### DISCUSSION THREAD

**Discussion by rasfarrf5** [rxhanson/Rectangle#409] [Ideas]
Can we have same shortcut for similar resize window operation.

For eg., 

**Existing Behaviour:**
`Option + Command + <- or ->`, will rearrange window to half. This works like a charm. 

**Additional Expected Behaviour:**
Now if we press again same shortcut i.e, `Option + Command + <- or ->`. Then it should resize window to 1/3rd of the screen size.

This will reduce minimal shortcut to remember for users and it increase developer usability.

**Comment by rxhanson** [comment]
Uncheck the box for moving to adjacent display on repeated shortcut execution.
